ironfish-cli/src/utils/asset.ts

```diff
@@ -124,9 +124,13 @@ export async function selectAsset(
   const choices = balances.map((balance) => {
```

Comment:
We should
- give verified status when selecting an asset
- put verified assets at top of list

(can do in future PR)